Parent/Parent.xcodeproj/project.pbxproj

```diff
@@ -1589,74 +1589,6 @@
 			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-defaults-Parent/Pods-defaults-Parent-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
-		61DA38861D468D2C002FEE31 /* Fabric */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			name = Fabric;
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "${PODS_ROOT}/Fabric/run 7cf76817b5c71690f9c8655cc366155508371fc6 f47067a1c45bff8852736f919c942fabe3f29421be87e8157e920616d5f4766b";
-		};
-		932411A92D00F2805358D87F /* [CP] Check Pods Manifest.lock */ = {
```

Comment:
I still see these two steps in the parent build phases, so I'm guessing these were either redundant steps, or steps for a target we have removed.